### PR TITLE
Update 0994 spool file

### PR DESCRIPTION
**Pull request by dmccurdybah**
## Description of change
Updated 0994 spool file to include "Enter the date you expect to be released from active duty" question.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#22359

## Things to know about this PR
- Tested locally and QA reviewed.
- Change is behind `edu_form_omb_and_expiration` flag

### CODE REVIEW

File: app/workers/education_form/templates/0994.erb [hunk 1]
```diff
@@ -30,12 +30,16 @@ Night Time Telephone Number: <%= @applicant.nightTimePhone %>
...
 
 <% if !Flipper.enabled?(:edu_form_omb_and_expiration) -%>
 Are you in the Selected Reserve or National Guard and do
 you expect to be called to duty for 30 days or more?: <%= yesno_or_blank(@applicant.activeDutyDuringVetTec) %>
+<% end -%>
+<% if Flipper.enabled?(:edu_form_omb_and_expiration) -%>
+Enter the date you expect to be released from active duty: <%= @applicant.expectedReleaseDate %>
+
 <% end -%>
 <% if Flipper.enabled?(:edu_form_omb_and_expiration) -%>
```
Comment (shawkey-daniel) [inline app/workers/education_form/templates/0994.erb:44]: Can't we just combine this?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]

File: spec/fixtures/education_benefits_claims/0994/kitchen_sink.json [hunk 2]
```diff
@@ -64,5 +64,6 @@
   "highestLevelofEducation": "other",
   "otherEducation": "Other Education Type",
   "privacyAgreementAccepted": true,
-  "expectedActiveDutyStatusChange": true
+  "expectedActiveDutyStatusChange": true,
+  "expectedReleaseDate": "2025-01-01"
 }
```
Comment (thilton-oddball) [inline spec/fixtures/education_benefits_claims/0994/kitchen_sink.json:69]: Not a big deal but you might go ahead and throw in a newline at the end here.

**Review by thilton-oddball — APPROVED** [review]
One comment but otherwise looks good.

File: spec/fixtures/education_benefits_claims/0994/kitchen_sink.json (hunk 2 above)
Comment (dmccurdybah) [reply spec/fixtures/education_benefits_claims/0994/kitchen_sink.json:69]: Sorry, missed this but I'll hit that soon

**Pull request merged**